PowerShell/JumpCloud Module/Private/settings/New-JCSettingsFile.ps1

```diff
@@ -0,0 +1,61 @@
+function New-JCSettingsFile {
+    [CmdletBinding()]
+    param (
+        [Parameter(
+            HelpMessage = 'To Force Re-Creation of the Config file, set the $force parameter to $tru'
+        )]
+        [switch]
+        $force
+    )
+
+    begin {
+        # Config should be in /PowerShell/JumpCloudModule/Config.json
+        $ModuleRoot = (Get-Item -Path:($PSScriptRoot)).Parent.Parent.FullName
+        $configFilePath = join-path -path $ModuleRoot -childpath 'Config.json'
+
+        # Define Default Settings for the Config file
+        $config = @{
+            'parallel' = @{
+                'Eligible'     = @{value = Get-JCParallelValidation; write = $false; copy = $true }
+                'Override'     = @{value = $false; write = $true; copy = $true }
+                'MessageCount' = @{value = 0; write = $true; copy = $true }
+                'HelpMessage'  = @{value = 'JumpCloud PowerShell Module now processes Get requests in parallel, to disable this functionailty run: Set-JCSettingsFile -parallelOverride $true'; write = $false; copy = $false };
+                'Calculated'   = @{value = $false; write = $false; copy = $true }
+            }
+            'updates'  = @{
```

Comment:
TODO: remove before we release, we are just testing with a new group of settings, not implemented